db-service/lib/cqn2sql.js

```diff
@@ -719,14 +734,36 @@ class CQN2SQLRenderer {
    */
   UPSERT(q) {
     const { UPSERT } = q
+    const entity = this.name(q.target?.name || UPSERT.into.ref[0])
+    const alias = UPSERT.into.as
```

Comment:
Move that line down to where it is required → principle of locality

Reply:
This has been bothering me for a while now. Is it really necessary to respect the `alias` for `INSERT`, `UPSERT` and `UPDATE` queries ? My understanding is that these aliases don't do anything except for making some  unit test pass.

These two queries would do the exact same. It is not even clear to me where this alias could possibly be consumed.
```
INSERT INTO sap_capire_bookshop_Books as Books (<columns>) <data>

INSERT INTO sap_capire_bookshop_Books (<columns>) <data>
```

If you agree I will take this opportunity to just ignore these root modifying query aliases.